web/sites/default/config/block.block.westy_views_block__content_list_block_assigned_as_a_contact.yml

```diff
@@ -0,0 +1,29 @@
+uuid: 80f27921-76fe-496f-8560-564bbc72a56b
```

Comment:
Instead of placing this as part of the block layout, could we add it to the Layout Builder layout? https://pe-671-employees.pantheonsite.io/admin/config/people/accounts/display/full

You may want to add a new column at the bottom of the layout and add it there, instead of adding it to the left column, because in my testing that made it too small and harder to use.

If you want to while you're at it, you could also remove the "Reports" block from the block layout and move it into the left column, so it doesn't push down the table and look odd:

![image](https://github.com/user-attachments/assets/93699709-3cd2-45b5-a1ae-d2c2a679c57b)

Reply:
Both items are now placed via layout builder. What's the advantage in your opinion to using layout builder instead of blocks? Also, I improved the display of user names in the Reports section.

Reply:
I guess in my mind the main advantage here specifically, is that the user account page was already built with Layout Builder, so having it all in one place is a bit cleaner than having it use both approaches.

But generally I find Layout Builder is much easier to use because: 1. you get a visual preview of what the layout will look like and a drag-and-drop builder with the ability to use multiple different layouts/sections, and 2. the layout is stored along with each entity type/view mode, instead of being just a global list of all the placed blocks, which IMO makes it much easier to maintain and understand.